Modules/xSharePoint/DSCResources/MSFT_xSPUserProfileProperty/MSFT_xSPUserProfileProperty.schema.mof

```diff
@@ -0,0 +1,75 @@
+
+/*
+**Description**
+
+This resource will create a property in a user profile service application. 
+It creates, update or delete a property using the parameters that are passed in to it .
+
+The parameter DisplayOrder is absolute. ie.: If you want it to be placed as the 5th field of section Bla, which has propertyName value of 5000 then your DisplayOrder needs to be 5005
+If no DisplayOrder is added then SharePoint adds it as the last property of section X.
+
+Length is only relevant if Field type is "String".
+
+This Resource  doesn't currently support removing existing user profile properties
+
+**Example**
+xSPUserProfileProperty WorkEmailProperty
+{
+    Name = "WorkEmail2"
+    UserProfileService = "User Profile Service Application"
+    DisplayName = "Work Email"
+    Type = "Email"
+    Description = "" #implementation isn't using it yet
+    PolicySetting = "Required"
+    PrivacySetting = "Everyone"
+    MappingConnectionName = "contoso.com"
+    MappingPropertyName = "mail"
+    MappingDirection = "Import"
+    Length = 10
+    DisplayOrder =25 
+    IsEventLog =$false
+    IsVisibleOnEditor=$true
+    IsVisibleOnViewer = $true
+    IsUserEditable = $true
+    IsAlias = $false 
+    IsSearchable = $false 
+    TermStore = ""
+    TermGroup = ""
+    TermSet = "" 
+    UserOverridePrivacy = $false
+}
+
+*/
+
+
+[ClassVersion("1.0.0.0"), FriendlyName("xSPUserProfileProperty")]
+class MSFT_xSPUserProfileProperty : OMI_BaseResource
+{
+    [Key] string Name ;
```

Comment:
@TravisEz13 We don't have descriptions on anything in xSharePoint yet - I'll add a backlog item to back capture this for all resources though